dev/nix/shell.nix

```diff
@@ -1,5 +1,16 @@
 with import <nixpkgs> {};
 
+# Pin the nixpkgs channel
```

Comment:
It would be good to add a comment explaining why you are pinning the nixpkgs channel (can refer to this PR).